.circleci/config.yml

```diff
@@ -403,6 +415,28 @@ jobs:
           coverage-reports: /tmp/workspace/ui/coverage/lcov.info
           project-token: $CODACY_PROJECT_TOKEN
 
+  api-contract-upload:
+    executor: pact-executor
+    steps:
+    - attach_workspace:
+        at: /tmp/workspace
+    - run:
+        name: Pact Broker Publish
+        working_directory: ~
+        command: |
+          pact-broker publish /tmp/workspace/pacts --broker-base-url "https://pact-broker.dsp-eng-tools.broadinstitute.org/" --consumer-app-version ${CIRCLE_SHA1} --branch ${CIRCLE_BRANCH}
```

Comment:
This uploads our generated contracts to Broad's Pact Broker.

Reply:
Does this use a credential, or is this an open endpoint?

Reply:
Good catch. It does. I added a clarifying comment.

I asked devops if I could post the path to the secret. Waiting to hear if that is allowed.

Reply:
I added a link to a Confluence article with the details instead of posting the details in the comment directly.